<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LXG-PC" name="127.0.0.1:62025" tests="1" failures="1" timestamp="7 Aug 2016 09:35:22 GMT" time="86.276" errors="0">
  <testcase name="comment" time="66.85" classname="cn.crazy.appium.testcases.MaiMaiTest">
    <failure type="java.lang.AssertionError" message="expected object to not be null">
      <![CDATA[java.lang.AssertionError: expected object to not be null
	at cn.crazy.appium.page.MaiMaiPage.comment(MaiMaiPage.java:27)
	at cn.crazy.appium.testcases.MaiMaiTest.comment(MaiMaiTest.java:38)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- comment -->
</testsuite> <!-- 127.0.0.1:62025 -->
